### PR TITLE
Pol 563 Make Azure policies ready for spend recommendations

### PR DESCRIPTION
**Pull request by lawrethao**
### Description

Updates fields for azure cost policies to make them ready for spend recommendations

### Issues Resolved

These changes resolve issues with field mappings in certain Azure cost policy templates

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/azure/idle_compute_instances/azure_idle_compute_instances.pt [hunk 1]
```diff
@@ -491,29 +497,50 @@ EOS
...
       field "id" do
         label "ID"
+        path "resourceID"
+      end
+      field "accountName" do
+        label "Account Name"
+        path "subscriptionName"
+      end
+      field "accountID" do
+        label "Account Id"
```
Comment (jim-slattery-rs) [inline cost/azure/idle_compute_instances/azure_idle_compute_instances.pt:542]: Maybe capitalize:
```suggestion
        label "Account ID"
```

File: cost/azure/idle_compute_instances/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
 
+## v3.0
+
+- applying data normalization updates for spend recommendations api. this change breaks current iterations expecting specific output types being pushed.
+- fixed issue with potential nil pointer type error issue
+- updated savings field to round to 3rd decimal
+- Normalizing fields for recommendations:
+  - updated 'tags' field to slice of string values
+  - Renamed 'id' to 'resourceID'
+  - Added 'AccountID' field with subscriptionID value
```
Comment (jim-slattery-rs) [inline cost/azure/idle_compute_instances/CHANGELOG.md:11]: Looks like the field is named like this:
```suggestion
  - Added 'accountID' field with subscriptionID value
```

File: cost/azure/idle_compute_instances/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+  - Added 'AccountName' field with subscriptionName value
```
Comment (jim-slattery-rs) [inline cost/azure/idle_compute_instances/CHANGELOG.md:12]: Looks like the field is named liked this:
```suggestion
  - Added 'accountName' field with subscriptionName value
```

File: cost/azure/idle_compute_instances/CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+  - Changed 'savings' to be a plain number
+  - Added 'savings_currency' to record the currency symbol.
```
Comment (jim-slattery-rs) [inline cost/azure/idle_compute_instances/CHANGELOG.md:14]: Just noticed this field is different -- maybe we should switch it to use camel-case?
```suggestion
  - Added 'savingsCurrency' to record the currency symbol.
```

Comment (lawrethao) [reply cost/azure/idle_compute_instances/CHANGELOG.md:14]: Id prefer that, I was mirroring the AWS ones but I was looking to probably get those changed as well if you have no issues with it.

File: cost/azure/unused_sql_databases/azure_unused_sql_databases.pt [hunk 5]
```diff
@@ -70,6 +70,8 @@ credentials "azure_auth" do
   tags "provider=azure_rm"
 end
 
+auth "auth_rs", type: "rightscale"
```
[the remark this answers is not in the document]
Comment (rshade) [reply cost/azure/unused_sql_databases/azure_unused_sql_databases.pt:73]: just adding `type: rightscale` its not supported in EMEA. I can show you out to adjust it when we merge this.

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**